CHANGELOG.md

```diff
@@ -43,6 +43,14 @@ We use *breaking* word for marking changes that are not backward compatible (rel
 This release also disables gossip mode by default for all components.
 See [this](docs/proposals/approved/201809_gossip-removal.md) for more details.
 
+:warning: This release moves Thanos docker images and artifacts to Golang 1.12. This release include change in GC's memory release which gives following effect (source: https://golang.org/doc/go1.12):
+
+> On Linux, the runtime now uses MADV_FREE to release unused memory. This is more efficient but may result in higher reported RSS. The kernel will reclaim the unused data when it is needed. To revert to the Go 1.11 behavior (MADV_DONTNEED), set the environment variable GODEBUG=madvdontneed=1.
+
+To ensure correct memory reporting:
+* Use memory heap allocation metric exposed by Golang.
```

Comment:
Maybe it would be worth mentioning that it is `go_memstats_heap_alloc_bytes`

Reply:
And maybe `use` -> `look at` or `observe`

Reply:
Thanks Man I need your memory (: